web/client/components/map/openlayers/swipe/SliderSwipeSupport.jsx

```diff
@@ -36,7 +36,13 @@ const VSlider = ({ type, map, widthRef }) => {
     }, [ type ]);
 
     useEffect(() => {
-        widthRef.current = map.getProperties().size[0] / 2;
+        const callback = (event) => {
```

Comment:
After an investigation on the compare behaviour the issue seems related to the devicePixelRatio.

These are the values printed after changing the display settings of the pc to 125%:

![image](https://user-images.githubusercontent.com/19175505/110305621-2c342380-7ffd-11eb-8a69-aa81c40bae2d.png)

The map canvas still uses the original screen width 1920px because it computes the size width multiply by the device pixel ratio so probably the solution is to multiply the getWidth() inside the EffectSupport by the pixel ratio.

https://github.com/geosolutions-it/MapStore2/blob/f7698720254b4028afbf78cffde6b7a4002cf2bd/web/client/components/map/openlayers/swipe/EffectSupport.jsx#L29
```js
const width = getWidth() * map.pixelRatio_;
```
At this point we should also fix the getHeight();

So please remove this callback and restore the previous code in this file.